profiling/examples/simple.rs

```diff
@@ -32,6 +32,16 @@ fn some_inner_function(_iteration_index: usize) {
     burn_time(10);
 }
 
+fn function_scope_function(){
```

Comment:
There's some (missing) whitespace in places that makes me think cargo fmt hasn't been run

Reply:
I am still seeing places that have no space before the opening { brace. Is Github's view just stale?